### PR TITLE
add support for Alert service

### PR DESCRIPTION
**Pull request by BoTranVan**

### CODE REVIEW

File: alert.go [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+}
+
+// HTTPHeaders is ...
+type HTTPHeaders struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// LoadBalancers is ...
+type LoadBalancers struct {
```
Comment (greatbn) [inline alert.go:91]: Use another name? Maybe this name will be confused with Loadbalancer service?

File: alert.go [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+		ReceiverID string   `json:"receiver_id"`
+		Methods    []string `json:"methods"`
+	} `json:"receivers"`
+	ResourceType string    `json:"resource_type"`
+	UserID       string    `json:"user_id"`
+	Volumes      []Volumes `json:"volumes,omitempty"`
+}
+
+// AutoScale is ...
+type AutoScale struct {
```
Comment (greatbn) [inline alert.go:254]: ditto

File: alert.go [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+}
+
+// Instances is represents instances payload - which servers will be monitored
+type Instances struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+// Volumes is represents volumes payload - which volumes will be monitored
+type Volumes struct {
```
Comment (greatbn) [inline alert.go:78]: Same LoadBalancers. I think we should use `Alarm` prefix for these structs ?

**Comment by BoTranVan** [comment]
change to branch support-autoscale-alert-manager

**Pull request closed without merging**